charts/hlf-k8s/README.md

```diff
@@ -10,6 +10,13 @@ Hlf-k8s runs Hyperledger Fabric v1.4.
 
 - Kubernetes 1.14+
 
+## Changelog
+
+### 1.5.0
+
+- `appChannel` changed to `appChannels` (list)
+- `appChannel.name` changed to `appChannels[].channelName`
+
```

Comment:
Could you move that into a CHANGELOG.md file ? Also having an UPGRADE.md could be nice

Reply:
will do